### ISSUE TITLE
[Bug]: File Template not parsed in Code

### ISSUE THREAD

**Issue by Basti007** [laravel-idea/plugin#726]
### Bug description

I edited the file template `PHP File Header` and added it to the `Laravel Ideal View` in the `Code` tab.
The View template looks like this:
```php
<?php
#parse("PHP File Header.php")
?>
```

The file header looks like this:
```php
/**
 * ${FILE_NAME}
 *
 * @date      ${DAY}.${MONTH}.${YEAR}
*/

```

The code which is generated using the `Create View` option:
```php
<?php
#parse("PHP File Header.php")
?>
```

Might this be a PHPStorm bug?
This feature already worked in a previous release, it broke at some release (I used it before).

### Plugin version

6.3.0.223

### Operating system

Windows

### Steps to reproduce

1. Go to Editor > File and Code Templates > Code Tab
2. Add a `parse` directive to Laravel Ideal View
3. Try generating a view

**Comment by adelf** [comment]
Reproduced. I'll try to fix. Thank you.

**Comment by adelf** [comment]
Fixed in the 6.3.1. Thanks
